integration_tests/dbt_project/models/exposures.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+
+exposures:
+  - name: customers
+    label: CustomersFTW
+    type: dashboard
+    maturity: high
+    url: https://bi.tool/dashboards/1
+    description: >
+      Did someone say "exponential growth"?
+
+    depends_on:
+      - ref('customers')
+
+    owner:
+      name: Callum McData
+      email: data@jaffleshop.com
+    meta:
+      referenced_columns:
+        - name: id
+          data_type: numeric
+          source: ref('customers')
```

Comment:
I think that maybe instead of "source" we should call it "node"?
Since it can contain either models or sources.